### PR TITLE
Add composer.json to support installing with composer

### PR DESCRIPTION
**Pull request by acoulton**
We're keen to use this module and it would be great to have it via composer/packagist if possible. I noticed there's no explicit licence - I've said BSD-3 in the composer but obviously change that if you would rather do something else. Also listed you and Jeremy as authors in line with the README.

### CODE REVIEW

**Comment by hkdobrev** [comment]
:+1:

**Comment by zeelot** [comment]
Sorry this slipped through my inbox, I'll pull this in tomorrow and add it to packagist. I don't think I want this installed as a kohana module though (making the same changes to the core modules composer files) since it's perfectly fine for these to end up in `vendor/`.

**Comment by zeelot** [comment]
Do you want to make that change? Otherwise I can do it tomorrow as well.

Cheers.

**Comment by acoulton** [comment]
@Zeelot thanks, I've updated the composer.json to be a library rather than a kohana-module.

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+		{
+			"name":     "Jeremy Lindblom",
+			"email":    "jeremeamia@gmail.com",
+			"homepage": "https://github.com/jeremeamia",
+			"role":     "developer"
+		}
+	],
+	"support": {
+		"issues":   "https://github.com/Zeelot/kohana-dependencies/issues",
+		"source":   "https://github.com/Zeelot/kohana-dependencies/issues"
```
Comment (zeelot) [inline composer.json]: Is this right? Shouldn't it point to the github repo and not the issues page?

**Comment by acoulton** [comment]
@Zeelot sorry about that typo, fixed now.

**Comment by zeelot** [comment]
Thanks! Added to packagist: https://packagist.org/packages/zeelot/kohana-dependencies

**Comment by acoulton** [comment]
Thanks @Zeelot

**Pull request merged**